catalog/app/utils/yaml.js

```diff
@@ -0,0 +1,10 @@
+import yaml from 'js-yaml'
+
+export default function parseYaml(inputStr) {
+  try {
+    return yaml.safeLoad(inputStr, 'utf8')
+  } catch (error) {
+    console.error(error)
+    return ''
```

Comment:
do we really need an empty string here? smth like `null` or even `undefined` seems more appropriate imo